prebid-server/endpoints/openrtb2/pbs-endpoint-auction.md

```diff
@@ -110,14 +110,49 @@ This is a corresponding sample response to a sample OpenRTB 2.5 bid request:
   }
 }
 ```
-### OpenRTB Fields
+### OpenRTB Request Fields
 
-Prebid Server accepts all OpenRTB 2.5 fields and passes them in the request to all bid and analytics adapters. Some fields are processed by Prebid Server in the following ways:
+Prebid Server accepts all OpenRTB 2.5 fields and passes them in the request to all bid and analytics adapters. Some ORTB 2.6 fields are supported. Here are the fields with special processing:
 
-#### Currency
+{: .table .table-bordered .table-striped }
+| ORTB Field | Version | Notes |
+| --- | --- | --- |
+| id | 2.5 | if host config generate-storedrequest-bidrequest-id config is true
+    if $.id is not set, generate a random value
+    if the storedrequest is from AMP or from ext.prebid.storedrequest, then replace any existing $.id with a random value
+if $.id contains "{{UUID}}", replace that macro with a random value |
+| source.tid | 2.5 | if source.tid is not set:
+   set source.tid to a random UUID
+if host config generate-storedrequest-bidrequest-id config is true
+    if the storedrequest is from AMP or from a top-level stored request (ext.prebid.storedrequest), then replace any existing $.source.tid with a random value
+if $.source.tid contains "{{UUID}}", replace that macro with a random value |
```

Comment:
For PBS-Go, there is a host config parameter ``auto_gen_source_tid` (defaults to true) which must be enabled for the TID logic.